share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -157,6 +159,24 @@ function(rocm_install_targets)
                         NAMELINK_ONLY
             )
         endif()
+        if(T_TYPE MATCHES ".*_LIBRARY" AND ROCM_SYMLINK_LIBS AND NOT ROCM_CREATED_${PROJECT_NAME}_SYMLINK)
+            set(ROCM_CREATED_${PROJECT_NAME}_SYMLINK TRUE CACHE INTERNAL "Record symlink created.")
+            string(TOLOWER "${PROJECT_NAME}" LINK_SUBDIR)
+            if(CMAKE_HOST_WIN32)
+                set(SYMLINK_CMD "file(COPY \${LINK_TGT} DESTINATION \${LINK_NAME})")
+            else()
```

Comment:
I would suggest we go directly to new structure on windows as don't have significant downstream users and for windows .dll the single path addition is much simpler.